crates/bevy_mesh/src/bounding.rs

```diff
@@ -13,21 +13,16 @@ use bevy_reflect::prelude::*;
 /// occupied by this entity, with faces orthogonal to its local axis.
 ///
 /// This component is notably used during "frustum culling", a process to determine
-/// if an entity should be rendered by a [`Camera`] if its bounding box intersects
-/// with the camera's [`Frustum`].
+/// if an entity should be rendered by a `Camera` if its bounding box intersects
+/// with the camera's `Frustum`.
 ///
-/// It will be added automatically by the systems in [`CalculateBounds`] to entities that:
-/// - could be subject to frustum culling, for example with a [`Mesh3d`]
+/// It will be added automatically by the systems in `CalculateBounds` to entities that:
+/// - could be subject to frustum culling, for example with a `Mesh3d`
 ///     or `Sprite` component,
-/// - don't have the [`NoFrustumCulling`] component.
+/// - don't have the `NoFrustumCulling` component.
 ///
 /// It won't be updated automatically if the space occupied by the entity changes,
-/// for example if the vertex positions of a [`Mesh3d`] are updated.
-///
-/// [`Camera`]: crate::camera::Camera
-/// [`NoFrustumCulling`]: crate::view::visibility::NoFrustumCulling
-/// [`CalculateBounds`]: crate::view::visibility::VisibilitySystems::CalculateBounds
-/// [`Mesh3d`]: crate::mesh::Mesh
+/// for example if the vertex positions of a `Mesh3d` are updated.
 #[derive(Component, Clone, Copy, Debug, Default, Reflect, PartialEq)]
 #[reflect(Component, Default, Debug, PartialEq)]
 pub struct Aabb {
```

Comment:
All of these bounding primitives should be in bevy_render, not bevy_mesh, since they're a rendering implementation detail that meshes shouldn't need to really care about. It'd also make these docs referencing cameras and various other components make sense.

`Mesh::compute_aabb` is the only method I see here that uses `Aabb`, but that could be handled by an extension trait or just a function in `bevy_render`. Is there some other reason why these types are currently here?

Reply:
i felt they should eventually end up in bevy_math, so i popped them out of bevy_render to begin